pkg/image/builder.go

```diff
@@ -101,6 +101,13 @@ func GetPrivateImages(upstreamDir string, checkedImages map[string]ImageInfo, al
 			if err != nil {
 				return err
 			}
+
+			// Skip charts folders containing subcharts.  The kustomization for subcharts will be defined
+			// inside of those charts, and should not be referenced at the parent chart level.
+			if info.IsDir() && info.Name() == "charts" {
```

Comment:
Can you check if this is needed only if useHelmInstall flag is set?

if info.IsDir() && info.Name() == "charts"  && (useHelmInstall[info.Name()] == true) {

Reply:
The check below this one is `if info.IsDir() && (useHelmInstall[info.Name()] == true)` , but it wasn't excluding subchart folders correctly.  Adding the useHelmInstall check would replicate the existing behavior.